includes/unsupported-theme-handlers/class-sensei-unsupported-theme-handler-cpt.php

```diff
@@ -229,4 +233,13 @@ public function force_page_template( $template ) {
 		return $template;
 	}
 
+	/**
+	 * Check if current theme is a FSE block one and if the current post type is a lesson.
+	 *
+	 * @return bool
+	 * @access private
+	 */
+	public function is_lesson_cpt_in_block_fse_theme() {
```

Comment:
This could use an `@since` too.

Reply:
Should we? It has `@access private`.

Reply:
Yeah, but I'm wondering why it's `public` in the first place since we only use it in this function.

Reply:
Oh, yep, that's a good question :)

Reply:
Ah, do we normally only add `@since` for public functions? I think I tend to add it for all new functions, and the WordPress documentation standards don't seem to suggest otherwise. 🤷🏻‍♀️

Reply:
I've made it private here https://github.com/Automattic/sensei/pull/7044/commits/771025e6e9eff24efc6effb16e0a7f631389c0e8. Added a `since` anyway :p